Sources/BottomSheetCalculator.swift

```diff
@@ -5,34 +5,39 @@
 import UIKit
 
 struct BottomSheetCalculator {
+    static let handleHeight: CGFloat = 20
+
     /// Calculates offset for the given content view within its superview, taking preferred height into account.
     ///
     /// - Parameters:
     ///   - contentView: the content view of the bottom sheet
     ///   - superview: the bottom sheet container view
     ///   - height: preferred height for the content view
     static func offset(for contentView: UIView, in superview: UIView, height: CGFloat) -> CGFloat {
-        let handleHeight: CGFloat = 20
         var targetHeight = contentHeight(for: contentView, in: superview, height: height)
 
         if height == .bottomSheetAutomatic {
-            targetHeight += handleHeight
+            targetHeight += BottomSheetCalculator.handleHeight
         }
 
-        return max(superview.frame.height - targetHeight, handleHeight)
+        return max(superview.frame.height - targetHeight, BottomSheetCalculator.handleHeight)
     }
 
     static func contentHeight(for contentView: UIView, in superview: UIView, height: CGFloat) -> CGFloat {
```

Comment:
@shredlocker Btw, do we want `handleHeight` to be part of the content height in `frameOfPresentedViewInContainerView` where we use this function? Because this is the total height of the bottom sheet only for automatic heights.

Reply:
@vadymmarkov No, handle height is not part of the content height in `frameOfPresentedViewInContainerView`.

Reply:
I guess we should subtract `handleHeight` when height is not `.bottomSheetAutomatic` to get the correct content height.

Reply:
That's true, or we should re-think our public API a bit. Now this array of target heights `[.bottomSheetAutomatic, 500]` would mean: use automatic height plus handle height for the first state and 500 is the total bottom sheet height (including handle) for the second state. Another option would be to think of 500 as the content height, but then it's arguably less predictable to the end user when the bottom sheet height ends up being 520. We could ofc rename `targetHeights` into `contentHeights` if we go with the second approach. Or do you like how it is now? @shredlocker